goldens/public-api/core/core.d.ts

```diff
@@ -163,11 +163,11 @@ export declare interface ConstructorSansProvider {
 export declare type ContentChild = Query;
 
 export declare interface ContentChildDecorator {
-    (selector: Type<any> | Function | string, opts?: {
+    (selector: Type<any> | InjectionToken<any> | Function | string, opts?: {
```

Comment:
Any reason not to use `unknown` instead of `any`? Historically we used to use `any` in our public api surface whenever the type was unknown, but ever since TypeScript introduced the `unknown` type we started to to use that one instead. Sometimes `unknown` is not suitable, but in this case it might be. Can you please try it?


```suggestion
    (selector: Type<any> | InjectionToken<unknown> | Function | string, opts?: {
```

Reply:
good idea! no strong reason. I just did it that way to align with existing generics for query decorators.  Updated it to use `unknown`!

I assume we can clean up the other generics too. I can try sending a follow-up, or incorporate this here, if possible.